apps/web/src/components/event-card/index.tsx

```diff
@@ -100,10 +140,16 @@ async function SignupQuotas({
 
   const isSingleQuota = quotas.length === 1;
 
+  const isQuota = !(
+    quotas.length === 1 &&
+    quotas[0].size === 20 &&
+    quotas[0].signupCount === 0
+  );
```

Comment:
Let's have the check be a null check for `event.registrationEndDate` and `event.registrationStartDate` fields. And again move it one level higher.